app/views/artists/appointments/_show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h3><%= appointment_date %></h3>
+<table>
+  <% appointments.each do |appointment| %>
+    <tr>
+      <td><%= appointment.description %></td>
+      <td><%= link_to 'Edit', "/users/appointments/#{appointment.id}/edit", remote: true %></td>
```

Comment:
same here, please change this line to

`<td><%= link_to 'Edit', edit_artists_appointment_path(appointment.id), remote: true %></td>`